app/src/main/java/com/nextcloud/talk/adapters/messages/IncomingTextMessageViewHolder.kt

```diff
@@ -171,6 +171,7 @@ class IncomingTextMessageViewHolder(itemView: View, payload: Any) : MessageHolde
     }
 
     private fun processParentMessage(message: ChatMessage) {
+        val maxReplyLength = 250
```

Comment:
I would make this a constant so it is only instantiated once since that is basically what it is.